cmd/bbolt/command_check.go

```diff
@@ -37,10 +84,16 @@ func checkFunc(cmd *cobra.Command, dbPath string) error {
 	}
 	defer db.Close()
 
+	var opts []bolt.CheckOption
+	opts = append(opts, bolt.WithKVStringer(CmdKvStringer()))
```

Comment:
nit: I think this could be simplified as:

```golang
opts := []bolt.CheckOption{bolt.WithKVStringer(CmdKvStringer())}
```

Reply:
👍🏽